RadarSDK/RadarAPIClient.m

```diff
@@ -37,6 +37,19 @@ - (instancetype)init {
     return self;
 }
 
++ (NSDictionary *)headersWithPublishableKey:(NSString *)publishableKey {
+    return @{
+        @"Authorization": publishableKey,
+        @"Content-Type": @"application/json",
+        @"X-Radar-Config": @"true",
+        @"X-Radar-Device-Make": [RadarUtils deviceMake],
+        @"X-Radar-Device-Model": [RadarUtils deviceModel],
+        @"X-Radar-Device-OS": [RadarUtils deviceOS],
+        @"X-Radar-Device-Type": [RadarUtils deviceType],
+        @"X-Radar-SDK-Version": [RadarUtils sdkVersion]
```

Comment:
are we omitting `User-Agent` now? is that to preserve the original header from the device?

Reply:
After discussing with @tjulien, we want to keep the original `User-Agent` (which can contain useful info, especially on Android), and send this info in separate headers instead